Resources/views/Sitemap/index.html.twig

```diff
@@ -1,7 +1,7 @@
 <ul class="cmf-sitemap">
     {% for url in urls %}
         <li>
-            <a href="{{ url.location }}" title="{{ url.label }}">{{ url.label }}</a>
+            {% if url.depth is not null %} <span class="indent-{{ url.depth }}"></span>{% endif %}<a href="{{ url.location }}" title="{{ url.label }}">{{ url.label }}</a>
```

Comment:
i am no frontend expert but i would have expected the class to be on the `<li>` element rather than a span. if span, it should probably wrap the `<a>` rather than just be before it for cleaner styling options.